src/librados/librados_c.cc

```diff
@@ -1010,18 +1010,19 @@ extern "C" int _rados_ioctx_pool_stat(rados_ioctx_t io,
   ls.push_back(pool_name);
 
   map<string, ::pool_stat_t> rawresult;
-  err = io_ctx_impl->client->get_pool_stats(ls, rawresult);
+  bool per_pool;
```

Comment:
nit: suggest to initialize this with something - just to avoid possibility to **random** [mis]behavior if get_pool_stats works improperly one day